packages/@okta/vuepress-site/docs/guides/terraform-configure-device-signin-standards/main/index.md

```diff
@@ -0,0 +1,383 @@
+
+    * `disk_encryption_type`: The set of required encryption settings for the device. The only option is `ALL_INTERNAL_VOLUMES` ( all internal volumes are encrypted).
+
+    * `screenlock_type`: A set of required screen lock actions and behaviors.  The supported options are `BIOMETRIC` (biometric locking) and `PASSCODE` (passcode locking).
+
+Your policy can also check signals on devices from a supported third-party signal provider, such as the Google Chrome Device Trust Connector in the Chrome browser. Specify its device signals in your Terraform code with the fields with the prefix `tpsp_`. For example, `tpsp_os_version` instead of `os_version`.  Okta for device assurance for ChromeOS requires using [Workforce Identity Adaptive Multi-factor Authentication (AMFA)](https://www.okta.com/learn/adaptive-mfa/). Developer Edition Okta orgs don't support the Google Chrome Device Trust Connector by default. If you're a paying customer, contact Okta developer support to enable third-party signal support on a Developer Edition Okta org.
+
+The following code shows a typical macOS device assurance policy:
+
+```hcl
+resource okta_policy_device_assurance_macos macos_example {
+  name = "macOS example"
+  os_version = "12.4.6"
+  disk_encryption_type = toset(["ALL_INTERNAL_VOLUMES"])
+  secure_hardware_present = true
+  screenlock_type = toset(["BIOMETRIC", "PASSCODE"])
+}
+```
+
+### Windows
+
+Create a Windows device assurance policy:
+
+1. Add the `okta_policy_device_assurance_windows` resource to your configuration.
+
+1. Set its `name` argument to a user-visible name for this policy.
+
+1. Set arguments for different signals. The following are the most commonly checked signals. For the complete signal list, see the [provider documentation for this resource](https://registry.terraform.io/providers/okta/okta/latest/docs/resources/policy_device_assurance_windows).
+
+    * `os_version`: The minimum operating system for a device.
+
+    * `disk_encryption_type`: The set of required encryption settings for the device. The only option is `ALL_INTERNAL_VOLUMES` ( all internal volumes are encrypted).
+
+    * `screenlock_type`: A set of required screen lock actions and behaviors. The supported options are `BIOMETRIC` (biometric locking) and `PASSCODE` (passcode locking).
+
+Your policy can also check signals on devices from a supported third-party signal provider, such as the Google Chrome Device Trust Connector in the Chrome browser. Specify its device signals in your Terraform code with the fields with the prefix `tpsp_`. For example, `tpsp_os_version` instead of `os_version`.  Okta for device assurance for ChromeOS requires using [Workforce Identity Adaptive Multi-factor Authentication (AMFA)](https://www.okta.com/learn/adaptive-mfa/). Developer Edition Okta orgs don't support the Google Chrome Device Trust Connector by default. If you're a paying customer, contact Okta developer support to enable third-party signal on a Developer Edition Okta org.
```

Comment:
```suggestion
Your policy can also check signals on devices from a supported third-party signal provider, such as the Google Chrome Device Trust Connector in the Chrome browser. Specify its device signals in your Terraform code with the fields with the prefix `tpsp_`. For example, `tpsp_os_version` instead of `os_version`.  Okta for device assurance for ChromeOS requires using [Workforce Identity Adaptive Multi-factor Authentication (AMFA)](https://www.okta.com/learn/adaptive-mfa/). Developer Edition orgs don't support the Google Chrome Device Trust Connector by default. If you're a paying customer, contact Okta developer support to enable third-party signal on a Developer Edition org.
```